website/content/docs/installing/no-gen-resources.mdx

```diff
@@ -0,0 +1,466 @@
+
+<Tabs>
+<Tab heading="CLI">
+
+```bash
+$ boundary scopes create -name 'org' -scope-id 'global' \
+  -recovery-config /tmp/recovery.hcl \
+  -skip-admin-role-creation \
+  -skip-default-role-creation
+<truncated>
+
+$ boundary scopes create -name 'project' -scope-id <org_scope_id_from_last_step> \
+  -recovery-config /tmp/recovery.hcl \
+  -skip-admin-role-creation \
+  -skip-default-role-creation
+<truncated>
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_scope" "org" {
+  scope_id    = "global"
+  name        = "organization"
+  description = "Organization scope"
+
+  auto_create_admin_role   = false 
+  auto_create_default_role = false
+}
+
+resource "boundary_scope" "project" {
+  name                     = "project"
+  description              = "My first project"
+  scope_id                 = boundary_scope.org.id
+  auto_create_admin_role   = false 
+  auto_create_default_role = false
+}
+```
+
+</Tab>
+</Tabs>
+
+
+
+#### Create an Auth Method
+
+Create an auth method in the organization scope.
+
+<Tabs>
+<Tab heading="CLI">
+
+```bash
+$ boundary auth-methods create password \
+  -recovery-config /tmp/recovery.hcl \
+  -scope-id <org_scope_id> \
+  -name 'my_method' \
+  -description 'My password auth method'
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_auth_method" "password" {
+  name        = "my_password_auth_method"
+  description = "Password auth method"
+  type        = "password"
+  scope_id    = boundary_scope.org.id
+}
+```
+
+</Tab>
+</Tabs>
+
+#### Create a Login Account
+
+Create a login account for the auth method.
+
+<Tabs>
+<Tab heading="CLI">
+
+```bash
+$ boundary accounts create password \
+  -recovery-config /tmp/recovery.hcl \
+  -login-name "myuser" \
+  -password "foofoofoo" \
+  -auth-method-id <auth_method_id_from_last_step"
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_account" "myuser" {
+  name           = "myuser"
+  description    = "User account for my user"
+  type           = "password"
+  login_name     = "myuser" 
+  password       = "foofoofoo"
+  auth_method_id = boundary_auth_method.password.id
+}
+```
+
+</Tab>
+</Tabs>
+
+#### Create a User
+
+Create a user and associate the user with the login account created in the previous step.
+This user will also be the principal in the role we create in the following step.
+
+<Tabs>
+<Tab heading="CLI">
+
+```bash
+$ boundary users create -scope-id <org_scope_id> \
+  -recovery-config /tmp/recovery.hcl \
+  -name "myuser" \
+  -description "My user!"
+
+$ boundary users add-accounts -id <myuser_user_id> -account <myuser_account_id>
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_user" "myuser" {
+  name        = "myuser"
+  description = "My user!"
+  account_ids = [boundary_account.myuser.id]
+  scope_id    = boundary_scope.org.id
+}
+```
+
+</Tab>
+</Tabs>
+
+
+#### Create Roles to Manage Scopes
+
```

Comment:
It might be worth some text here explaining to the user that these four roles will end up granting the same access as if role autogeneration had been enabled, but with Terraform in full control?